pkg/config/config.go

```diff
@@ -479,6 +480,11 @@ type RuntimeConfig struct {
 	// Option to disable hostport mapping in CRI-O
 	// Default value is 'false'
 	DisableHostPortMapping bool `toml:"disable_hostport_mapping"`
+
+	// Option to specify the timezone inside the container.
+	// If it's empty, the container will use the same timezone
+	// as the host machine but in UTC.
```

Comment:
The host might have some local time zone set. So it will use whatever this time zone would be. The `UTC` (or `Etc/UTC`) here is a bit confusing.

Reply:
I removed that line completely to avoid confusion.